### DISCUSSION TITLE
Used to work but stopped!

### DISCUSSION THREAD

**Discussion by greekislan** [eliemichel/MapsModelsImporter#251] [Q&A]
The error I get from RenderDoc 1.19 is "Error kicking capture" "Error injecting into process xxxx for capture". I'm running the same versions of blender, renderdoc and mapsmodelsimporter and don't get why I'm getting this error - any ideas please? Thanks

**Comment by Abderrezakmerabet** [comment]
Same here and even if i manage to capture something using renderdoc v1.24 blender can't read the RDC file stating : 

Invalid RDC capture file. Please make sure that:
1. You are using the recommended RenderDoc Version for this AddOn
   - RenderDoc Version 1.5 - 1.9 for MapsModelsImporter <= 0.3.2
   - RenderDoc Version = 1.10 for MapsModelsImporter >= 0.3.3 and <= 0.3.7
   - RenderDoc Version = 1.13 for MapsModelsImporter >= 0.4.0
2. You are importing from Google Maps or Google Earth web
3. You were MOVING in the 3D view while taking the capture (you can use the "Capture after delay"-button in RenderDoc).

Before opening a new Issue on GitHub please download a working sample file to check if this works on your Computer.
Please be patient. If there's no error message it might still be loading. 
It can take a minute or two to load it and Blender will get unresponsive during this time.
Find sample files here: https://github.com/eliemichel/MapsModelsImporter-samples

If it works with a sample file you most probably shouldn't open a new issue on GitHub but figure out how to use RenderDoc.
Find instructions about using RenderDoc by searching YouTube for "Capturing Google Maps with RenderDoc"

If the sample file doesn't work:
Please report to MapsModelsImporter developers providing the full console log with debug information.
First turn on debug output by activating the "Debug Info"-checkbox under Edit > Preferences > Add-ons > MapsModelsImporter
On Windows systems console log is accessible in Windows > Toggle System Console (right click to copy).
On Linux systems you have to run Blender from the console to get the debug output

**Reply by eliemichel** [reply]
Given the error message, you are not using the last version of this add-on. And if you'd do, you'd read in the error message that you need RenderDoc 1.19. Newer versions will **not** work!

edit: Okey, I was able to reproduce the problem of OG in RD1.19, I'm on it

**Reply by Abderrezakmerabet** [reply]
I actually made it work, it works perfectly with this configuation : 3.3 blender - 1.19 renderdoc - 0.5.0 MapsModelsImporter et edge

**Comment by eliemichel** [comment]
You can now update to RenderDoc 1.25 with the last version of this add-on: https://github.com/eliemichel/MapsModelsImporter/releases/tag/v0.6.0

I was able to reproduce your issue on RD1.19 and with RD1.25 it is working again!

**Reply by wsq1501** [reply]
Unfortunately, it seems as though it is still not working, following a fresh install of RD 1.25.
The main issue is the error "Error injecting into process XXXX for capture". This is using Chrome, same with Edge.
No matter what I try, even with RD 1.19, it is the same error each time.
I cannot remember as there are no separate instructions but does RenderDoc require inserting of a file from MapsModelsImporter V.0.6.0?
I thought MMI was only used as a plugin add-on for Blender, is this correct? Therefore RD 1.25 install should remain as original. 
Thanks

**Reply by eliemichel** [reply]
There is indeed nothing particular to inject in RD. If you still cannot inject, maybe check that both RD and Chrome run on the same graphics device (e.g. if you're on a dual GPU laptop)

**Comment by MikeLerg-BCI** [comment]
I and others I work with are having a similar issue, where they can get the combination of apps and tools to work on the first try, but after that, it won't complete a capture... I have gotten past that issue following your latest instructions, but the resulting RDC file does not contain 3D Meshes... on your TWITTER account, I have sent a detailed message with the RDC and all related information as well as a screen-grabbed video of me conducting the capture using RenderDoc. I am able to import any of your sample RDC files into Blender... 

I'm sure I must be doing something very simple but wrong... I look forward to your input once you've had a chance to review the materials I sent to you on Twitter.

**Reply by eliemichel** [reply]
Replied on twitter, this was indeed a bug and lead to v0.6.1!
